tests/table_retrieval_task_basic_test.py

```diff
@@ -90,38 +88,39 @@ def test_tp_updating(self):
         self.assertIsInstance(fetaqa_results, TaskResultsDataModel)
         self.assertIsInstance(retr_perf, RetrievalPerformanceDataModel)
         self.assertIsInstance(downs_perf, DownstreamTaskPerformanceDataModel)
-        self.assertEqual(
-            retr_perf.model_dump(),
-            {"k": 2, "accuracy": 0.5, "precision": None, "recall": None},
-        )
+        performance_dict = retr_perf.model_dump()
+        self.assertIn("k", performance_dict)
+        self.assertEqual(2, performance_dict["k"])
+        self.assertIn("avg_retrieval_duration_wall_clock", performance_dict)
+        self.assertIn("avg_retrieval_duration_process", performance_dict)
+        self.assertIn("retrieval_duration_wall_clock", performance_dict)
+        self.assertIn("retrieval_duration_process", performance_dict)
+        self.assertIn("accuracy", performance_dict)
+        self.assertEqual(0.5, performance_dict["accuracy"])
```

Comment:
Fails with `FAILED tests/evaluators/evaluator_basic_test.py::TestEvaluator::test_basic_run_task - AssertionError: 0.5 != 0.0`

Reply:
resolved, after converting db ids from int to strings, test no longer works